src/applications/financial-status-report/components/AvailableDebts.jsx

```diff
@@ -1,8 +1,7 @@
 import React, { useEffect } from 'react';
 import { connect } from 'react-redux';
-import Telephone, {
-  CONTACTS,
-} from '@department-of-veterans-affairs/component-library/Telephone';
+import Telephone from '@department-of-veterans-affairs/component-library';
+import { CONTACTS } from '@department-of-veterans-affairs/component-library/contacts';
```

Comment:
You might wanna go ahead and migrate the `Telephone` components in this file to `va-telephone`.

Reply:
Weird, not sure how I managed to change that import since I wasn’t concerned w/ that application. That was an accident w/ searching, by bad. I'll update the component while I'm at it tho!

Reply:
I've just removed those changes for now.